sklearn/tests/test_impute.py

```diff
@@ -230,7 +230,7 @@ def test_imputation_most_frequent():
 
 def test_imputation_most_frequent_objects():
     # Test imputation using the most-frequent strategy.
-    for marker in (None, np.nan, "NAN", "", 0):
```

Comment:
Why not test for `None` anymore? Some users might want to use `None` as the missing value marker. I think it's good to make sure our code works properly in that case.